.github/workflows/ci.yml

```
@@ -42,7 +42,7 @@ jobs:
 
     strategy:
       matrix:
-        ember-version:
+        ember-version: 
```

Somehow added a trailing space to this unchanged line?

```suggestion
        ember-version:
```